cprnc_py/netcdf/netcdf_variable_scipy.py

```diff
@@ -47,11 +47,33 @@ def _get_data_from_slices(self, dim_slices):
         dim_slices should match the dimensionality of this variable
         """
 
-        # FIXME(wjs, 2015-12-31) Will this work on scalar data?
         if len(dim_slices) > 0:
-            vardata = self._var[dim_slices].copy()
+            # TODO(mfd, 2016-09-19) Find a better way of doing this
```

Comment:
Can you describe the case(s) you're trying to handle with this new code? (My apologies if you already described this to me before, and I'm forgetting.) My initial reaction is to ask if we should really be handling these cases - i.e., it seems that it may actually be correct to raise an exception in these cases rather than trying to handle them. But I don't fully understand what you're doing here, so would be happy to understand this better.